content/blog/2022-1-18-January-22-heartbeat.md

```diff
@@ -0,0 +1,352 @@
+---
+title: January '22 Heartbeat
+date: 2022-01-18
+description: >
+  Monthly updates are here! You will find great tutorials and workflows from  
+  the Community, Online course is now open, decision making strategies
+  for  MLOps tools, and more! Welcome to 2022!🎇
+
+descriptionLong: |
+  This month you will find:
+    
+    🥰 Tutorials and workflows from the Community,
+
+    🗣 Upcoming Events,
+
+    📰 Data Science and AI News,
+
+    🧐 MLOps tool decision strategies,
+
+    😎 GitHub Awesomeness,
+
+    💻 Online Course is live,
+
+    🚀 Info on our growing team, and more!
+picture: 2022-01-18/Frame 61draft.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/january-22-heartbeat/1025
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - DVC Studio
+  - Heroku
+  - Reproducibility
+  - Stateful training
+  - SQL
+  - Git
+---
+
+# From the Community
+
+Happy New Year! Hope you got some good R&R and stayed healthy at the end of
+2021, because 2022 has lots of great things in store!
+
+![Heartbeat!](https://media.giphy.com/media/7ILAGpJWoQYWA0j60C/giphy.gif)
+
+## Diego Jardim - MLOps: A Complete Hands-On Introduction
+
+[In Part 1](https://poatek.com/2021/12/20/mlops-a-complete-and-hands-on-introduction-part-1/)
+of his two part series,
+[**Diego Jardim**](https://www.linkedin.com/in/diegosevero/) of
+[Poatek](https://poatek.com/) takes us through the basics of MLOps and the
+stages of implementation and maturity of an MLOps pipeline. He closes by
+introducing us to some tools to help a team progress through these stages, which
+include DVC and CML.
+
+[In Part 2](https://poatek.com/2021/12/29/mlops-a-complete-and-hands-on-introduction-part-2/)
+he delves into more detail and code on how to set up version control of
+everything with DVC and automation of experimentation and reporting with CML.
+Finally he uses FastAPI and Herouku for model serving and deployment. You can
+find all the scripts for the project in
+[this GitHub repository.](https://github.com/dsjardim/fraud-detection-mlops)
+
+<external-link 
+href="https:/https://poatek.com/2021/12/29/mlops-a-complete-and-hands-on-introduction-part-2/"
```

Comment:
Small typo here with `https:/`

Reply:
Thanks Roger!

Reply:
@rogermparent  I corrected this and it's not showing that it changed.  
![image](https://user-images.githubusercontent.com/34289199/149840253-817cea27-4aca-4b64-b202-c5c3a368b2fa.png)

Reply:
Nvrmnd - file problem.  Resolved.